### PR TITLE
Testjournal Script Refactor

### PR DESCRIPTION
**Pull request by gabrielngdvsa**
## Description

Test Journal Refactor 

## Checklist

- [ ] PR title includes the JIRA ticket number and an understandable description
- [ ] Branch is rebased against the latest develop
- [ ] Code has been tested manually
- [ ] PR link added to JIRA ticket

### CODE REVIEW

File: src/element-references/des-element-references.json [hunk 1]
```diff
@@ -14,5 +14,13 @@
     "re-key-test-btn": "#rekey-search-button",
     "partial-practise-mode-btn": "#practice-mode-test-report-button",
     "full-practise-mode-btn": "#practice-mode-full-button"
+  },
+  "test-center-journal": {
```
Comment (turnerc1) [inline src/element-references/des-element-references.json:18]: Something minor, but are you able to add "des-" at the start of this please. We will try and follow like the dashboard please.

**Review by turnerc1 — CHANGES_REQUESTED** [review]

**Review by Cullima — APPROVED** [review]

File: src/features/des/testcentreJournal.feature [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-        # Given I am signed in as a random "mobexaminer1" user
+        Given I am signed in as a random "mobexaminer1" user
         When I wait on element "des-dashboard::page-title" to be displayed
-        Then I wait on element "#test-centre-journal-button" to be displayed
-        When I click on the element "#test-centre-journal-button"
-        Then I wait on element "#test-title" to be displayed
-        Then I expect that element "#test-title" matches the text "Candidate search"
-        And I expect that element "#test-centre-location" matches the text "Example Test Centre"
-        And I add "Ali Campbell" to the inputfield "#typeahead-prevent-manual-entry"
+        Then I wait on element "test-center-journal::journal-button" to be displayed
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:6]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 3]
```diff
@@ -1,18 +1,18 @@
...
+        When I click on the element "test-center-journal::journal-button"
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:7]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
+        Then I wait on element "test-center-journal::page-title" to be displayed
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:8]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 5]
```diff
@@ -1,18 +1,18 @@
...
+        Then I expect that element "test-center-journal::page-title" matches the text "Candidate search"
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:9]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 6]
```diff
@@ -1,18 +1,18 @@
...
+        And I expect that element "test-center-journal::location-text" matches the text "Example Test Centre"
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:10]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 7]
```diff
@@ -1,18 +1,18 @@
...
+        And I add "Ali Campbell" to the inputfield "test-center-journal::candidiate-search-text"
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:11]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 8]
```diff
@@ -1,18 +1,18 @@
...
         And I click on the button with the text "Ali Campbell"
-        And I wait on element "#show-booking-button-text" to be enabled
-        And I click on the element "#show-booking-button-text"
-        And I expect that element "#candidate-no-booking-at-test-centre-tomorrow-message" matches the text "This candidate does not have a booking at Example Test Centre for tomorrow."
+        And I wait on element "test-center-journal::show-booking-button" to be enabled
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:13]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 9]
```diff
@@ -1,18 +1,18 @@
...
+        And I click on the element "test-center-journal::show-booking-button"
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:14]: Update to new page title (des-test-center-journal)

File: src/features/des/testcentreJournal.feature [hunk 10]
```diff
@@ -1,18 +1,18 @@
...
+        And I expect that element "test-center-journal::candidiate-booking-message" matches the text "This candidate does not have a booking at Example Test Centre for tomorrow."
```
Comment (turnerc1) [inline src/features/des/testcentreJournal.feature:15]: Update to new page title (des-test-center-journal)

**Review by turnerc1 — CHANGES_REQUESTED** [review]

**Review by turnerc1 — APPROVED** [review]

**Pull request merged**